storage/src/vespa/storage/storageserver/mergethrottler.h

```diff
@@ -187,6 +186,16 @@ class MergeThrottler : public framework::Runnable,
 
     bool onSetSystemState(const std::shared_ptr<api::SetSystemStateCommand>& stateCmd) override;
 
+    /*
+     * When invoked, merges to the node will be BUSY-bounced by the throttler
+     * for a configurable period of time instead of being processed.
+     *
+     * Thread safe, but must not be called if _stateLock is already held, or
+     * deadlock will occur.
+     */
+    void apply_timed_backpressure();
+    bool backpressure_mode_active() const;
```

Comment:
This is only intended for tests? Could duplicate code with merge_is_backpressure_throttled() be removed and/or method implementations placed next to each other?

Reply:
Only for tests, as test would otherwise depend on specific reply behavior of `MergeThrottler` when it's in this mode. No problem to depend on this for `MergeThrottlerTest` cases, but felt like leaking details when done for a disparate error handler test. I'll de-dupe the code that does not depend on mutex protection.